### DISCUSSION TITLE
How to block ads apart from using cosmetic filters?

### DISCUSSION THREAD

**Discussion by luxysiv** [JingMatrix/ChromeXt#81] [Show and tell]
Can I use Eruda Console to block ads url request? Do you think about it?

**Comment by JingMatrix** [comment]
It is better for you to find/write UserScripts that can do the job.
You may need to know [CSP rules](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src).

**Comment by luxysiv** [comment]
I was read it but not understand. Do you have an example userscript ?

**Comment by JingMatrix** [comment]
Check this one: https://github.com/JingMatrix/ChromeXt/issues/33#issuecomment-1501221989

**Comment by luxysiv** [comment]
Ok I will think about it
